src/pkg/packaging/rpm/package.targets

```diff
@@ -0,0 +1,271 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Import Project='package.props' />
+
+  <UsingTask TaskName="ReplaceFileContents" AssemblyFile="$(LocalBuildToolsTaskDir)core-setup.tasks.dll"/>
+  <UsingTask TaskName="BuildFPMToolPreReqs" AssemblyFile="$(LocalBuildToolsTaskDir)core-setup.tasks.dll"/>
+
+  <PropertyGroup>
+    <IsRPMBasedDistro Condition="$(PackageTargetRid.StartsWith('rhel')) or
```

Comment:
I'm not sure if the plan is to generate RPMs on OpenSuse as well, but that is also RPM based.